lib/tasks/dev.rake

```diff
@@ -0,0 +1,33 @@
+namespace :dev do
+  require 'factory_bot_rails'
+  require 'faker'
+
+  desc 'Generate 100 developer record'
+  task :generate_data, [:developers_num] => :environment do |_, args|
+    set_args(args)
+
+    %w(ruby php java react).each { |name|
```

Comment:
Doesn't `rubocop` warn to use `do ~ end` instead of `{ }` for a block with multiple lines?
This is written here at least.
https://github.com/bbatsov/ruby-style-guide#single-line-blocks